Added Walkspeed Bypass and Anti Fling

Hizon2492 opened the pull request:
added new walkspeed bypass that uses cframe and isn't as easy to detect as most
fixed antifling and made it more performant and better (it isnt only "haha go through players" now) < id add notifications for this so you know who's flinging but idk how annoying that would be

Hizon2492 committed:
Update Source

added new walkspeed bypass that uses cframe and isn't as easy to detect as most
fixed antifling and made it more performant and better (it isnt only "haha go through players" now) < id add notifications for this so you know who's flinging but idk how annoying that would be

Hizon2492 committed:
Added Velocity Check for anti fling

Hizon2492 committed:
Turned into connection and added unwalkspeedbypass

Hizon2492 committed:
Fixed walkspeedbypass (for real this time)

Hizon2492 committed:
Removed Entire Character Collision on Anti Fling

Due to me still being flinged cuz unknown reasons by people's head i decided to make entire character uncollideable in the case they're detected flinging

Hizon2492 committed:
Remade NoVoid

now skids cant kill u haha

Hizon2492 committed:
Update Source

Hizon2492 commented:
Forgot to add in I made it so less methods of voiding you work, also changed antifling a bit so it's less relaxed

Hizon2492 committed:
Update Source

Hizon2492 committed:
Update Source

minionon6 commented:
> Forgot to add in I made it so less methods of voiding you work, also changed antifling a bit so it's less relaxed

I have a question.

Hizon2492 commented:
> I have a question.

when are you asking it?

Hizon2492 committed:
Platform detection "patch"

in reality i just changed parent, you can probably add a instance detection hook for it.
